azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/state/AppReduxState.kt

```diff
@@ -23,7 +23,7 @@ internal class AppReduxState(displayName: String?) : ReduxState {
                 device = AudioDeviceSelectionStatus.RECEIVER_SELECTED,
             ),
             videoStreamID = null,
-            displayName = displayName
+            displayName = "$displayName(me)"
```

Comment:
I think (me) will be localized in future, we can move to view class.

Reply:
addressed